### DISCUSSION TITLE
Theme Discussion

### DISCUSSION THREAD

**Discussion by AmanNegi** [AmanNegi/FoodLink#8] [Q&A] [answered]
Discuss regarding Theme here.

**Comment by AmanNegi** [comment, marked as answer]
![image](https://github.com/AmanNegi/FoodLink/assets/37607224/bd2a169b-3eba-42dc-8797-9a2ffc6fa4cf)

This looks good to me, let me know. @iSundhararajan

**Reply by iSundhararajan** [reply]
Yep, it works :)

**Comment by AmanNegi** [comment]
For Fonts we can use: https://www.fontpair.co/pairings/ibm-plex-sans-condensed-ibm-plex-sans

Easily available at GoogleFonts as well.
